JapaneseFilter/sections/specific.txt

```diff
@@ -3,6 +3,7 @@
 !
 ! /banner/banner_ - if it is necessary, make this rule domain specific
 !
+korehakituk.nekonikoban.org##body > div[style]
```

Comment:
better `body > div[style="text-align: center;"]`

Reply:
There are two placeholders, the top one and the bottom. Mine covers both.